src/components/AcceptAll.tsx

```diff
@@ -29,16 +29,21 @@ export function AcceptAll({
   };
 
   return (
-    <div className="column-content">
+    <div className="column-content" role="none">
       <div>
         <div>
-          <p id="consent-dialog-title" className="text-title text-title-left">
+          <p
+            id="consent-dialog-title"
+            role="heading"
```

Comment:
prob a silly question but when should we add the role attribute?

Reply:
When the role is not already implicit in the tag. For instance, an h2 will automatically have a heading role, and a button will have a button role. From what I understand, it's usually best to use the proper tag, because sometimes you get added benefits that don't come with the role alone. But sometimes there are reasons to mix it up.

Reply:
so i guess in this instance we're using a <p> as a header, so a role is needed for screen readers, etc to know this is being used as a header. i'll keep role in mind when not using the proper tag. thanks annalisa!!!

Reply:
Yeah, that makes sense. 😅